insights/parsers/foreman_log.py

```diff
@@ -89,3 +89,24 @@ class CandlepinErrorLog(LogFileOutput):
         '2016-09-07 16:49:24,650 [=, org=] WARN  org.apache.qpid.transport.network.security.ssl.SSLUtil - Exception received while trying to verify hostname'
     """
     pass
+
+
+@parser('foreman-ssl_access_ssl.log')
```

Comment:
In the master branch, the usage of `specs` is updated to `datasource`:

you should import it at first, like:
```
from insights.specs import foreman_ssl_access_ssl_log
...
@parser(foreman_ssl_access_ssl_log)
```
See line-31~35

Reply:
@xiangce : Changes made as per suggestions. Please review it